src/platforms/react-native/touchevents/index.mdx

```diff
@@ -52,7 +52,7 @@ Each touch event is automatically logged as a breadcrumb and displays on the das
 
 ## Tracking Specific Components
 
-You can let Sentry know which components to track specifically by passing the `sentry-label` prop to them. If Sentry detects a component with a `sentry-label` within a touch's component tree, it will be logged on the dashboard as having occurred in that component. If we cannot find a component with the label, we will fall back to the `accessibilityLabel` prop.
+You can let Sentry know which components to track specifically by passing the `sentry-label` prop to them. If Sentry detects a component with a `sentry-label` within a touch's component tree, it will be logged on the dashboard as having occurred in that component. If we cannot find a component with the label, we will fall back to the specified `labelName` prop if specified or else we use `displayName`.
```

Comment:
Is it worth showing how to have accessibilityLabel passed automatically to LabelName?
By passing the touchEventBoundaryProps in the Sentry.Warp fn?
```
Sentry.wrap(App, {
  touchEventBoundaryProps: {
    labelName: 'accessibilityLabel',
  },
}
```

Reply:
Yes, definitely, it's in the options below https://github.com/getsentry/sentry-docs/pull/8540/files#diff-3d1b4536b38174c093a7782ff39a496ecd1e1dfaf037f0c527675a28cca3d3baR87

Reply:
@daniel-johns-99 Thanks for mentioning this, I'll add one more example.